src/components/common/header/Notification/Notification.tsx

```
@@ -1,34 +1,24 @@
+import { getSendAlarm } from '../../../../api/alarm.api';
```

ESLint

[error] 'getSendAlarm' is defined but never used.

(@typescript-eslint/no-unused-vars)